python/simple/v1/app.py

```diff
@@ -0,0 +1,80 @@
+from ampli import *
+from plugins import MyEventIDPlugin, SegmentPlugin
+from util import get_api_key
+
+api_key = get_api_key('.env', 'API_KEY')
+write_key = get_api_key('.env', 'WRITE_KEY')
+
+# Initialize the Ampli instance with LoadOptions and LoadClientOptions
+ampli.load(LoadOptions(Environment.DEVELOPMENT, False,
+                       client=LoadClientOptions(api_key=api_key, configuration=Config(server_zone='EU'))))
+
+# Identify using IdentifyProperties in tracking plan
+ampli.identify("user_id", Identify(required_number=16.6, optional_array=['abc', 'test']))
+
+# Group Identify using GroupProperties in tracking plan
+ampli.group_identify("Org", "Engineer", Group(required_boolean=True, optional_string=None))
+
+# Set groups for user
+ampli.set_group("user_id", "Org", ["Engineer", "DevOp"])
+
+# track strongly typed event class with ampli.track
+ampli.track("user_id", EventNoProperties(), event_options=EventOptions(user_id="user_id", device_id="device_id"))
```

Comment:
When passing user_id in EventOptions it probably makes sense to call with `ampli.track(None, EventNoProperties(), event_options=...)`